packages/fxa-payments-server/src/components/PaymentForm/index.scss

```diff
@@ -95,6 +77,10 @@ form.payment {
   }
 }
 
+.blurb-addendum {
+  padding-top: 5px;
+}
+
```

Comment:
I noticed the font color, size, line height and weight for these links differ from the [MVP designs](https://www.figma.com/file/cOIAuHIIbrPbAlAIcpejNM/%F0%9F%8E%A1MVP-Subscription-Optimization-Flow?node-id=400%3A2098); can you make sure they match?